roles/openstack_helm_magnum/tasks/main.yml

```diff
@@ -358,15 +371,6 @@
               nodeSelector:
                 openstack-control-plane: enabled
 
-- name: Create Ingress
```

Comment:
Is this accidentally removed?

Reply:
no, there were 2 duplicated tasks

Reply:
Are you sure?  One is for registry and one is for magnum..

Reply:
yes, i am sure. there were 2 ingresses for magnum before